synapse/handlers/sync.py

```diff
@@ -1470,6 +1586,15 @@ def _generate_room_entry(self, sync_result_builder, ignored_users,
             full_state=full_state
         )
 
+        summary = None
+        if (
+            sync_config.filter_collection.lazy_load_members() and
+            any(ev.type == EventTypes.Member for ev in batch.events)
```

Comment:
This seems to not handle the case of an initial sync where `batch.events` may contain no membership events.

Reply:
oops, fixed.